plugin/helpers_config.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/utils"
+)
+
+// CheckRequiredServerConfiguration checks if the server is configured according to
```

Comment:
This can just be 
```suggestion
// CheckRequiredServerConfiguration implements Helpers.CheckRequiredServerConfiguration
```

Otherwise we will end up with two sources of truth.